internal/pkg/cli/app_show_test.go

```diff
@@ -332,10 +346,10 @@ Environments
 
 Workloads
 
-  Name    Type
-  ----    ----
-  my-svc  lb-web-svc
-  my-job  Scheduled Job
+  Name    Type           Environments
+  ----    ----           ------------
+  my-svc  lb-web-svc     test,prod
+  my-job  Scheduled Job  test,prod
```

Comment:
Let's also include an example where the service is not deployed at all to see if the table still makes sense in that case.

Reply:
added test case for the services which are not being deployed yet